tensorrt_llm/_torch/modules/linear.py

```
@@ -820,6 +820,191 @@ def load_weights_fused_gate_up_linear(self, module: Linear,
+              bias: Optional[torch.Tensor]):
+        alpha = module.alpha
+        if input.dtype != torch.float8_e4m3fn:
+            if module.input_scale is not None and not module.force_dynamic_quantization:
+                # Static quantization
+                fp8_input, _ = torch.ops.tensorrt_llm.static_quantize_e4m3_per_tensor(
+                    input, module.input_scale)
+            else:
+                # Dynamic quantization
+                fp8_input, input_scale = torch.ops.tensorrt_llm.quantize_e4m3_per_tensor(
+                    input)
+                alpha = model.weight_scale_2 * input_scale.to(torch.float32)
```

Ruff (0.12.2)

Undefined name `model`

(F821)